atlas-search/using-atlas-search.mdx

```diff
@@ -0,0 +1,157 @@
+    nodes {
+      streetAddress
+      contactName
+      workOrderNumber
+    }
+  }
+}
+```
+
+## Configuration
+
+For users who have more specific functionality needs, Atlas Search provides a series of advanced configurations to give you the most control on your data search. To manage these configurations, navigate to the 'Search Config' tab of Atlas Search.
+
+### Search Method
+
+Atlas Search provides two methods for searching - `stemming` and `fuzzy` - which can be used to tune the search results for your needs.
+
+#### High Relevancy with Stemming Search
+
+Stemming is the process of reducing a word to its root form or base word. This is the default search method of Atlas Search. Using stemming can increase search relevancy as it will search for both exact matches and matches on stemmed words.
+For example, a search for the word `running` will return results where running is in the queried field first but it will also return results that include variations of the word `run`.
+
+#### Handling Typos with Fuzzy Search
+
+The fuzzy search method can be toggled in the Search Config menu as an alternative to stemming.
+
+Fuzzy search can be used to handle typos in a search query. With fuzziness enabled, a search query for "Hallo Warld" would return results for both "Hello" and "World".
+
+You can specify the per-word 'distance' setting for Fuzziness to either 1 or 2 on the slider, which determines how many letters can be off in each word. If you set the 'Fuzzy distance' to the maximum distance of 2, a search query of "Haalo Waald" would work as per the previous example, still yielding the results for both "Hello" and "World".
+
+_Note: The higher the fuzziness distance allowed the greater the performance & relevancy hit will be to your search experience._
+
+### Filtering And Boosting Searchable Content
+
+By default, Atlas Search will search all supported WordPress Data Type objects (Posts, Pages, CPTs, ACFs, ACMs, etc.) by all of their supported fields (string, number, boolean, and other WordPress Data Type objects).
+
+The search will return results where each field-match to the search query term(s) are given an equal weighting (1), thus the objects with most search-match occurrences will rise to the top. In some cases this may not be the desired behavior. For example:
+
+- It may not be correct to consider some specific fields as searchable.
+- Searches matching some fields should be considered more relevant than others.
+
+In these cases, Atlas Search provides both a field-level searchable toggle and weighting slider. The searchable toggle allows for the inclusion/exclusion of a field from the search. The field-level weighting slider permits the configuration of some fields to be considered more relevant than others when determining the order of search results. The higher the weighting, the more relevant it is determined to be.
+
+![Atlas Search Plugin Config](./images/as-plugin-config.png)
```

Comment:
Might be helpful to include this image under each of the above config descriptions with an outline around the relevant UI component